data_processing.py

```diff
@@ -5,92 +5,185 @@
 import sys
 
 
-def create_csv_url(state, region, url="", table_num=0):
+def start_interface():
+    """Command line interface for scraping and/or inserting data into parking code database
+    Requires user input.
+    After typing state and region:  searches for existing .csv
+                                    searches for pdf
+                                    asks for url to scrape
 
-    region_file = region.replace(' ', '_')
+    URL scrape: scrapes the whole page
+                will ask user to identify table number (there will most likely be multiple tables in the html page)
+    Uses functions from scraper_functions.py
 
+    Insert into Database:   requires database.ini (holds database credentials)
+    Uses functions from database_functions.py
+
+    Returns: None
+
+    """
+
+    print("\n\n\nScraping Parking Requirements | Creating .csv | Inserting into Database\n")
+    state = input("Enter state (ex. NY, CA): ")
+    region = input("Enter region: ")
+
+    filename = region.replace(' ', '_')+"_"+state
```

Comment:
You repeat the same `os.path.join('input_csv', f'{filename}.csv')` twice. It'd be better to save that in this variable:

```suggestion
    _file_name = f"{region.replace(' ', '_')}_{state}.csv"
    file_path = os.path.join("input.csv", _file_name)
```

Then update the code to use `file_path`.